lte/gateway/python/magma/mobilityd/uplink_gw.py

```diff
@@ -132,7 +132,7 @@ def update_ip(
             vlan_id: vlan of the GW, None in case of no vlan used.
```

Comment:
Same for the changes in this files. We catch the error so we can add a `type: ignore`.